service/controller/v1/key/key.go

```diff
@@ -20,3 +25,7 @@ func CrtPath(certificateDirectory, clusterID string) string {
 func KeyPath(certificateDirectory, clusterID string) string {
 	return certPath(certificateDirectory, clusterID, "key")
 }
+
+func NginxPodProxyUrlTemplate(masterPrefix string, clusterID string) string {
+	return fmt.Sprintf("https://%s.%s:443/api/v1/namespaces/kube-system/pods/${1}:10254/proxy/metrics", masterPrefix, clusterID)
```

Comment:
the `masterPrefix` magic is there because all of the test has `apiserver.ID` instead of `master.ID` and after an hour of playing with test, a was unable to make it work.